main.mm

```diff
@@ -84,7 +84,7 @@ - (void)setup {
 
     // Load shaders.
     NSError *error = nil;
-    _library = [self.device newLibraryWithFile: @"shaders.metallib" error:&error];
+    _library = [self.device newLibraryWithFile: @"/Users/sonictk/Git/thirdparty/metal-without-xcode/shaders.metallib" error:&error];
```

Comment:
Sorry for the very very late replay, but can you remove this part? It exposes your local user name.

Reply:
Can confirm that this works with just `"shaders.metallib"`

Reply:
Hi @n-yoda, is it possible to add a permissive license like MIT for this repository's code? https://opensource.org/licenses/MIT My current employer will not allow me to contribute the simple fix for this if there is no appropriate licensing information together with the rest of the code in the repository. 

Thanks!